README.md

```diff
@@ -0,0 +1,38 @@
+# OpenActive Test Suite
+
+The general aim of this project is to allow end to end testing of the various flows and failure states.
+
+This repo hosts two different projects.
+* Broker service: this sits in between the test suite and the target site. This allows the test suite to watch for changes, and throws them back to it.
+* Test suite: this allows automated tests against the API
+
+## Configuration
+Configure the endpoint urls in:
+ - packages/openactive-broker-microservice/config/default.json
+    - bookingApiBase: URI for the booking service API
+    - openFeedBase: RPDE Feed URL
+ - packages/openactive-integration-tests/config/test.json
+    - bookingApiBase: URI for the booking service API
+    - microserviceApiBase: Broker URI
+
+# Installation
+ - `npm install`
+ - `npm run lerna bootstrap`
```

Comment:
Don't the changes in #7 mean this is part of install?

Reply:
Oops, hadn't spotted that :)

Reply:
Removed